berryio.json

```diff
@@ -5,15 +5,14 @@
     "description": "This app supports actions for APIs on the BerryIO project for the Raspberry Pi, such as GPIO status, get and set",
     "type": "information",
     "publisher": "Splunk Community",
-    "main_module": "berryio_connector.py",
-    "app_version": "2.0.3",
+    "main_module": "berryio_connector.pyc",
```

Comment:
this still needs to point to `.py`. builds will not contain the `.pyc` files, so install will fail otherwise